src/main/java/com/tw/go/plugin/BuildStatusNotifierPlugin.java

```diff
@@ -55,6 +58,7 @@ public BuildStatusNotifierPlugin() {
             Class<?> providerClass = Class.forName(properties.getProperty("provider"));
             Constructor<?> constructor = providerClass.getConstructor();
             provider = (Provider) constructor.newInstance();
+            templateName = properties.getProperty("settings.template");
```

Comment:
This IMO should come from the provider, to keep use of configuration files to a minimum.